quickstarts/microsoft.machinelearningservices/machine-learning-job-create-sweep-job/script/main.py

```diff
@@ -0,0 +1,109 @@
+# imports
```

Comment:
I think the folder should be named `scripts` - as mentioned in [best practices](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

Reply:
Yes, Done.